node/src/distributions/mod.rs

```diff
@@ -19,6 +19,7 @@ use self::mainnet::DistributionResult;
 use pallet_airdrop_claims::{MultiAddress, StatementKind};
 
 use tangle_primitives::{Balance, BlockNumber};
+use tangle_runtime::EXISTENTIAL_DEPOSIT;
```

Comment:
Can you pls import this from `tangle_primitives` so it will still compile with `--features testnet`

Reply:
Done.